CHANGELOG.md

```diff
@@ -7,7 +7,14 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+
+### Fixed
+
+- Allow plain objects to be returned from the `loadCallback` on `CustomSessionStorage` [#126](https://github.com/shopify/shopify-node-api/pull/126)
```

Comment:
Total nit: I think this is more of an addition than a fix, nothing was broken before.

Reply:
Fair point